Source/Hurl.Settings/Views/RulesetPage.xaml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Page
+    x:Class="Hurl.Settings.Views.RulesetPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Hurl.Settings.Views"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:model="using:Hurl.Library.Models"
+    xmlns:toolkit="using:CommunityToolkit.WinUI.Controls"
+    xmlns:uc="using:Hurl.Settings.Controls"
+    xmlns:vm="using:Hurl.Settings.ViewModels"
+    mc:Ignorable="d">
+    <Page.DataContext>
+        <vm:RulesetPageViewModel x:Name="ViewModel" />
+    </Page.DataContext>
+
+    <StackPanel MaxWidth="800" Spacing="16">
```

Comment:
Later, you might want to create a (custom) style for this to remove redundancy :) I suggest `MaxWidth="1000" Margin="24"`